### DISCUSSION TITLE
Error Message

### DISCUSSION THREAD

**Discussion by Tiger2Henschel** [Official-Husko/NN-Downloader#44] [Q&A] [answered]
Every time i try to download something from Multporn, it gives me this Error Message "[!] An error occured while downloading from multporn! Please report this. Exception: cannot access local variable 'raw_link' where it is not associated with a value". What does that mean or how can i fix that.

**Comment by Official-Husko** [comment, marked as answer]
Next time please open an issue ticket instead of a discussion page. Further support will be [provided in the issue ticket #45](https://github.com/Official-Husko/NN-Downloader/issues/45). Thank you.
